* strong aliasing between __environ and environ, when dynamiclly linking
against uClibc makes them get different addresses when busybox ash shell
is running, causing enviroment loading to fail

since uClibc writes to __environ,
but ash reads from environ.

diff -Nur uClibc-20060324.orig/libc/misc/internals/__uClibc_main.c uClibc-20060324/libc/misc/internals/__uClibc_main.c
--- uClibc-20060324.orig/libc/misc/internals/__uClibc_main.c	Fri Mar 24 10:12:06 2006
+++ uClibc-20060324/libc/misc/internals/__uClibc_main.c	Fri Mar 24 13:37:47 2006
@@ -118,7 +118,7 @@
  * environ symbol is also included.
  */
 char **__environ = 0;
-strong_alias(__environ,environ)
+weak_alias(__environ,environ)
 
 /* TODO: don't export __pagesize; we cant now because libpthread uses it */
 size_t __pagesize = 0;
